tests/test_roles.py

```diff
@@ -48,7 +48,7 @@ def test_data(self):
             name = role_dict.get("name")
             with self.subTest(name=name):
                 self.assertIsNotNone(name)
-                obj = Role.parse_obj(role_dict)
+                obj = Role.model_validate(role_dict)
```

Comment:
I cant judge this change.

Reply:
Its just because `parse_obj` method was deprecated in Pydantic 2.0 and the recommended replacement is `model_validate`.
The functionality still remains the same - the method converts the incoming dictionary into the Role object.